Modify search

gracewainaina opened the pull request:
Allow the user to search for a song in a song database

Commit:
Set playlist activity and modify song model

Designed view for playlist activity and modify song object to make it a Parse object as well

Commit:
Log out

Implement log out function so that user can log out

Commit:
Building search adapter and queue adapter

Built the search adapter for binding search results on search dialog.
Built the queue adapter for binding search results on the playlist

Commit:
Searchbar onquery listener text

Given a list of latest music, a user can select a song and initiate a search

Commit:
Attempt 2 [lost files] Searchbar onquery listener text

-Previous commit erased some files
-Given a list of latest music, a user can select a song and initiate a search

Commit:
Search view functionality completed

Users can now search

xuyueee commented:
It seems that your branch is still not rebased onto the master branch, since there's still a merge conflict.

Commit:
attempt to resolve conflict with master branch

attempt to resolve conflict with the master branch

Commit:
Resolving git comments on camel case, logs, constant, unused code

-Set up camel case variable names, eliminated unnecessary log statements, deleted commented out of use code
-Spotify remote app SDK and binary files updated when code was run

Commit:
Camel case

Set up camel case variables that were left out

Commit:
Merge conflict with master branch

Merge conflict with master branch

Commit:
syncing gradle in modify-search branch

syncing gradle in modify-search branch

Commit:
resolved conflict in modify-search branch

Commit:
unchecked change in search view in modify-search branch
